internal/tools/aws/database_multitenant.go

```diff
@@ -779,36 +857,73 @@ func (d *RDSMultitenantDatabase) connectRDSCluster(schema, endpoint, username, p
 	return closeFunc, nil
 }
 
-func (d *RDSMultitenantDatabase) createUserIfNotExist(ctx context.Context, username, password string) error {
-	_, err := d.db.QueryContext(ctx, "CREATE USER IF NOT EXISTS ?@? IDENTIFIED BY ? REQUIRE SSL", username, "%", password)
-	if err != nil {
-		return errors.Wrap(err, "failed to run create user SQL command")
+func (d *RDSMultitenantDatabase) ensureDatabaseIsCreated(ctx context.Context, databaseName string) error {
+	if d.databaseType == model.DatabaseEngineTypeMySQL {
+		// Query placeholders don't seem to work with argument database.
+		// See https://github.com/mattermost/mattermost-cloud/pull/209#discussion_r422533477
+		query := fmt.Sprintf("CREATE DATABASE IF NOT EXISTS %s CHARACTER SET ?", databaseName)
+		_, err := d.db.QueryContext(ctx, query, "utf8mb4")
+		if err != nil {
+			return errors.Wrap(err, "failed to run create database SQL command")
+		}
+	} else {
+		query := fmt.Sprintf("DROP DATABASE IF EXISTS %s", databaseName)
```

Comment:
Do we really want to drop the database if it exists? Wouldn't we just want to check if it exists and if it does then we don't do anything?

Is there any chance of hitting this on a database that has been used for a while?

Reply:
Good point. There shouldn't be a chance that we hit another database, but let's be safe and just check if it already exists instead.